tests/e2e/config/travis/wc-services-testing-helper.php

```diff
@@ -8,22 +8,6 @@
 Author URI: http://
 */
 
-define( 'JETPACK_DEV_DEBUG', true );
-define( 'WOOCOMMERCE_SERVICES_LOCAL_TEST_MODE', true );
-define( 'WOOCOMMERCE_CONNECT_FREQUENT_FETCH', true );
-define( 'WOOCOMMERCE_CONNECT_SERVER_URL', 'http://host.docker.internal:5000/' );
-define( 'WOOCOMMERCE_SERVICES_CI_TEST_MODE', true );
```

Comment:
Not needed when `WOOCOMMERCE_SERVICES_LOCAL_TEST_MODE` and `JETPACK_DEV_DEBUG` are already specified